crates/evm/src/evm/handler.rs

```diff
@@ -350,7 +350,10 @@ impl<SPEC: Spec, EXT: CitreaExternalExt, DB: Database> CitreaHandler<SPEC, EXT,
         let diff_size = calc_diff_size(context).map_err(EVMError::Database)? as u64;
```

Comment:
This can also be renamed to l1_diff_size, but thats a minor neat

Reply:
For now I only focused on structs and didn't get into the variable namings, because there are quite a lot of them.